src/main/java/eu/europeana/metis/sandbox/service/dataset/DatasetServiceImpl.java

```diff
@@ -89,6 +93,14 @@ public boolean isXsltPresent(String datasetId) {
     return datasetRepository.isXsltPresent(Integer.parseInt(datasetId)) != 0;
   }
 
+  @Override
+  public DatasetInfoDto getDatasetInfo(String datasetId) {
+    DatasetEntity datasetEntity = datasetRepository.findById(Integer.valueOf(datasetId)).orElseThrow(() -> new InvalidDatasetException(datasetId));
+    return new DatasetInfoDto(datasetId, datasetEntity.getDatasetName(), datasetEntity.getCreatedDate(), datasetEntity.getLanguage(),
+            datasetEntity.getCountry(), getHarvestingParameterDto(datasetId), isXsltPresent(datasetId));
+
```

Comment:
suggestion to remove this whitespace enter.

Reply:
Done